Dockerfile

```diff
@@ -1,16 +1,19 @@
-FROM node:8.1.4
+FROM node:8.9.0-alpine
```

Comment:
is this version of node installed on the servers?

Reply:
yes